usage/app/lib/Config.scala

```diff
@@ -25,6 +25,10 @@ object Config extends CommonPlayAppProperties with CommonPlayAppConfig {
 
   val defaultPageSize = 100
   val defaultMaxRetries = 6
+  val defaultMaxPrintRequestSizeInKb = 500
```

Comment:
minor: why not just do `val defaultMaxPrintRequestSizeInKb = 500 * 1024` here and get rid of `maxPrintRequestLength` above as `maxPrintRequestLength` feels like a config constant.